parm/config/gfs/config.fetch

```diff
@@ -0,0 +1,37 @@
+#! /usr/bin/env bash
+
+########## config.fetch ##########
+
+echo "BEGIN: config.fetch"
+
+# Get task specific resources
+source "${EXPDIR}/config.resources" fetch
+
+export ICSDIR="@ICSDIR@" # User provided ICSDIR; blank if not provided
+export BASE_IC="@BASE_IC@" # Platform home for staged ICs
+
+export FETCH_YAML_TMPL="${PARMgfs}/stage/master_gfs.yaml.j2"
+export fetch_yaml="stage_atm_cold.yaml"
+
+# Set ICSDIR (if not defined)
+if [[ -z "${ICSDIR}" ]] ; then
+
+  ic_ver="20240610"
+
+  if (( NMEM_ENS > 0 )) ; then
+    ensic="${CASE_ENS}"
+  fi
+
+  if [[ "${DO_OCN:-NO}" == "YES" ]] ; then
+    ocnic="mx${OCNRES}"
+  fi
+
+  export ICSDIR="${BASE_IC}/${CASE}${ensic:-}${ocnic:-}/${ic_ver}"
+
+fi
+
+# Use of perturbations files for ensembles
+export USE_OCN_ENS_PERTURB_FILES="NO"
+export USE_ATM_ENS_PERTURB_FILES="NO"
```

Comment:
All of the stage_ic variables should be removed.

```suggestion
export FETCH_YAML_TMPL="${PARMgfs}/fetch/C48_cold.yaml.j2"
```

Reply:
Made those updates.